src/nix/nix.md

```diff
@@ -48,12 +48,18 @@ manual](https://nixos.org/manual/nix/stable/).
 
 # Installables
 
+> **Warning** \
+> Installables are part of the unstable
+> [`nix-command`](@docroot@/contributing/experimental-features.md#xp-feature-nix-command)
+> [experimental feature](@docroot@/glossary.md#gloss-experimental-features),
```

Comment:
```suggestion
> [`nix-command` experimental feature](@docroot@/contributing/experimental-features.md#xp-feature-nix-command),
```

The two different link texts are almost indistinguishable from one. The exp feature listing is right there on the page that defines experimental features.